lib/api/core/statistics.js

```diff
@@ -145,6 +145,55 @@ module.exports = function (kuzzle) {
   };
 
   /**
+   * Gets the lasts saved statistics frame from a date
+   *
+   * @param requestObject
+   * @returns {Promise} promise resolved as a ResponseObject
+   */
+  this.getStats = function(requestObject) {
+    var
+      deferred = q.defer(),
+      response = {
+        statistics: {}
+      },
+      frames,
+      currentDate,
+      since;
+
+    currentDate = new Date().getTime();
+    since = requestObject.data.body.since;
+    if (since > currentDate) {
+      deferred.resolve(new ResponseObject(requestObject, response));
```

Comment:
quite agree with @jenow 
BadRequestError are for malformed queries (for example, missing a required paramereter, or wrong format).
Here, the request is well formed, but we just have no content to retrieve.